src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/DecisionOutcomeCallbackHandler.java

```diff
@@ -56,4 +61,30 @@ private CallbackResponse changeState(CallbackParams callbackParams) {
     public List<CaseEvent> handledEvents() {
         return EVENTS;
     }
+
+    private void deleteNotificationAndInactiveTasks(CaseData caseData, String authToken) {
```

Comment:
I see this kind of method being built a lot. Is there not a service / helper method we could put it in and call ?

Reply:
@ss-evoco  I have change the way to do that, now I'm overriding a method, please have a look